apps/frontend/src/pages/contributors/libs/helpers/get-contributor-columns/get-contributor-columns.helper.tsx

```diff
@@ -23,6 +27,15 @@ const getContributorColumns = (actions: {
 		header: "Projects",
 		size: 300,
 	},
+	{
+		cell: ({ row: { original: contributor } }) =>
```

Comment:
let's wrap this expression with curly braces

Reply:
hmm.. in the app we are using it like this.. not sure I understand